### PR TITLE
[PT] Improving the translation for concurrency lesson

### PR DESCRIPTION
**Pull request by elainenaomi**
Hi all! 👋 

I found some small issues in the Portuguese translation for the Concurrency Lesson.
Part of them I had to adapt the phrase to be more clear in Portuguese - Brazil because terms like `to match messages` can be a little confusing if we don't refer to `pattern matching`. 
Let me know if I have to create an issue or adapt a term to Portuguese - Portugal.

Thanks for this amazing project ✨ 

The examples from Elixir School helped a lot to start to understand Elixir, including the advanced topics 🙂

### CODE REVIEW

File: pt/lessons/advanced/concurrency.md [hunk 1]
```diff
@@ -1,19 +1,19 @@
 ---
-version: 1.1.1
+version: 1.1.2
```
Comment (thiamsantos) [inline pt/lessons/advanced/concurrency.md:2]:
```suggestion
version: 1.1.1
```
@elainenaomi Even though this is a improvement to the translation we should keep the same version as the original https://github.com/elixirschool/elixirschool/blob/master/en/lessons/advanced/concurrency.md. We only bump when the content from the original changes.

**Comment by thiamsantos** [comment]
@elainenaomi Thanks for the improvements, the text is much better :)

File: pt/lessons/advanced/concurrency.md (hunk 1 above)
Comment (elainenaomi) [reply pt/lessons/advanced/concurrency.md:2]: Oh! Ok! Thanks!

**Comment by thiamsantos** [comment]
Thanks @elainenaomi :heart: :tada:

**Pull request merged**